pkg/services/ngalert/notifier/receivers.go

```diff
@@ -210,15 +210,15 @@ func (am *Alertmanager) GetReceivers(ctx context.Context) apimodels.Receivers {
 		for _, integration := range rcv.Integrations() {
 			name := integration.Name()
 			sendResolved := integration.SendResolved()
-			lastNotify, lastNotifyDuration, lastNotifyError := integration.GetReport()
+			lastNotifyAttempt, lastNotifyAttemptDuration, lastNotifyAttemptError := integration.GetReport()
```

Comment:
This isn't important, but we could shorten these without losing much meaning because the fields in the struct are well named:

```suggestion
			ts, d, err := integration.GetReport()
```